docker-compose.yml

```diff
@@ -1,188 +1,197 @@
 version: '3.7'
```

Comment:
```suggestion
version: "3.7"
```

This is using " in every app, so we should use it here too.

Reply:
Sure, I missed this one 😁